doc/EFFECT_ON_CONDITION.md

```diff
@@ -172,3 +172,1082 @@ Other EOCs have some variables as well that they have access to, they are as fol
 | mutation: "activated_eocs" | { "this", `mutation_id` }
 | mutation: "processed_eocs" | { "this", `mutation_id` }
 | mutation: "deactivated_eocs" | { "this", `mutation_id` }
+
+
+## Character effects
+
+
+#### `u_mutate`, `npc_mutate`
+
+Your character or the NPC will attempt to mutate; used in mutation system, for other purposes it's better to use [`u_add_trait`](#`u_add_trait`, `npc_add_trait`)
+
+| Syntax | Optionality | Value  | Info |
+| --- | --- | --- | --- | 
+| "u_mutate" / "npc_mutate" | **mandatory** | int or [variable object](#variable-object) | one in `int` chance of using the highest category, with 0 never using the highest category |
+| "use_vitamins" | optional | boolean | default true; if true, mutation require vitamins to work | 
+
+##### Valid talkers:
+
+| Avatar | Character | NPC | Monster |  Furniture | Item |
+| ------ | --------- | --------- | ---- | ------- | --- | 
+| ✔️ | ✔️ | ✔️ | ❌ | ❌ | ❌ |
+
+##### Examples
+```json
+{ "u_mutate_category": 0 }
+```
+
+```json
+{ "npc_mutate_category": { "math": [ "1+1" ] }, "use_vitamins": false }
+```
+
+
+#### `u_mutate_category`, `npc_mutate_category`
+
+exactly same as u_mutate
```

Comment:
This one takes mutation category as a parameter so it's not the same



```suggestion
Similar to `u_mutate` but takes category as a parameter and guarantees mutation.

| Syntax | Optionality | Value  | Info |
| --- | --- | --- | --- | 
| "u_mutate_category" / "npc_mutate_category" | **mandatory** | string or [variable object](#variable-object) | mutation category |
| "use_vitamins" | optional | boolean | same as in `u_mutate` | 

##### Examples
```json
{ "u_mutate_category": "PLANT" }
{ "u_mutate_category": { "global_val": "next_mutation" }
---->> github eats these backticks ``` <----

```